### ISSUE TITLE
Redesign UX using Material

### ISSUE THREAD

**Issue by jerrytron** [choosatron/choosatron-write#9]
Once the basic features are functional in the current design / UX, I want to completely redo the design using Google Material guidelines, with sleek, intuitive interface. Here is an example for inspiration: https://atavist.com/cms/project/52749/sections/213603#details

**Comment by spleenboy** [comment]
The [material.angular](https://material.angularjs.org/latest/#/) package will help with this quite a bit.

**Comment by jerrytron** [comment]
Indeed! I'm excited about it. Watched the Google IO stuff and seems like there are some wonderful things we can use.

**Comment by Slade1972** [comment]
That atavist site looks nice. A sleek UI would be awesome, and as always, I'm happy to test things out, and build the software as required.
Reading that atavist site, it looks like drag and drop is supported from various "things" (other websites, files on the computer). Is this something that would be considered ? Or is that not possible?
